backend/prisma/schema.prisma

```diff
@@ -22,11 +22,35 @@ model event {
   description String
   image       String
   category    String
+  comments    comment[]
+  user        user     @relation(fields: [userId], references: [id])
+  userId      Int
+  date        String
+  time        String
+  lat         Float
+  long        Float
 }
 
 model user {
   id          Int      @id @default(autoincrement())
-  username    String
+  username    String   @unique
   email       String
   password    String
+  attendance  attendance[]
+  events      event[]
+}
+
+model comment {
+  id          Int      @id @default(autoincrement())
+  event       event    @relation(fields: [eventId], references: [id], onDelete: Cascade)
+  userPosting String
```

Comment:
Why not add reference to user?
Also why are we using username to refer user instead of user id, the primary key?

Reply:
**_To Do:_** add reference to user (Needed to delete comments)

_Why are we using username to refer user instead of user id, the primary key?_
To display the username in each comment. **_To Do_**: Mark username as unique in the data model